content/en/dynamic_instrumentation/_index.md

```diff
@@ -98,7 +99,7 @@ All probe types require the same initial setup:
 1. Go to the [Dynamic Instrumentation page][12].
 1. Click **Create Probe** in the top right, or click the three-dot menu on a service and select **Add a probe for this service**.
 1. If they are not prefilled, choose service, runtime, environment, and version.
-1. In the source code, specify where to set the probe by selecting either a class and method or a source file and line. If you set up Source Code Integration for your service, autocomplete shows suggestions for the selecting a file, and displays the file's code so you can choose the line.
+1. In the source code, specify where to set the probe by selecting either a class and method or a source file and line. If you opted into the [autocomplete and search public beta][17] and you set up Source Code Integration for your service, autocomplete shows suggestions for the selecting a class / method, and displays the file's code so you can choose the line.
```

Comment:
This sentence makes it sound like in order to get a better experience, you must BOTH enable "autocomplete and search private beta" AND set up source code integration. While there is added benefit for having both enabled, I wonder if we really need to spell it out here?

Reply:
Concretely, I propose to just remove the mention of source code integration from this paragraph.